src/models/Portfolio.ts

```diff
@@ -15,34 +43,46 @@ const PortfolioSchema = new Schema<
   IPortfolioUrl,
   Model<IPortfolioUrl>,
   IPortfolioUrl
->({
-  status: String,
-  submissionNo: Number,
-  portfolioUrl: {
-    type: String,
-    unique: true,
-  },
-  user: {
-    type: Schema.Types.ObjectId,
-    ref: 'User',
+>(
+  {
+    submissionNo: Number,
+    previousSubmissions: [
+      {
+        number: Number,
+        url: String,
+        date: Schema.Types.Date,
+      },
+    ],
+    resubmissionCount: Number,
+    portfolioUrl: {
+      type: String,
+      unique: true,
+    },
+    status: String,
+    reviewComments: [
+      {
+        author: String,
+        comment: String,
+        date: Schema.Types.Date,
+      },
+    ],
```

Comment:
I guess we can add validation here as well. Also didn't understood what's an author..?
```suggestion
    reviewComments: [
      {
        author: String,
        comment: { type: String, trim: true, minlength: 20 }
        date: Schema.Types.Date,
      },
    ],
```

Reply:
Adding validation here is a good idea! Hmmm, the Author needs a change in name. I intended to keep it as the comment the admin-reviewer would provide.